source/documentation/guidance/static-egress-ips.md

```diff
@@ -0,0 +1,9 @@
+## Static egress IP addresses
+
+In order to locate the GOV.UK PaaS static egress Internet Protocol (IP) addresses please [log in to your GOV.UK PaaS account](https://www.cloud.service.gov.uk/sign-in/) and go to the relevant link (depending on where your account is located):
+
+- [London static IP addresses](https://admin.london.cloud.service.gov.uk/support/static-ip)
```

Comment:
Should we say 'static egress' to be consistent?

Reply:
updated